main.go

```diff
@@ -52,13 +53,22 @@ func RunServer(logfile io.Writer) {
 	host := pflag.String("host", "0.0.0.0", "Host of the server")
 	tlsCertPath := pflag.String("tls-cert-path", "server.crt", "TLS Certificate path")
 	tlsKeyPath := pflag.String("tls-key-path", "server.key", "TLS Private key path")
+	watchmanHost := pflag.String("watchman-host", "192.168.122.1", "Watchman Metrics Host")
```

Comment:
The Agent is designed to work in various environments and to have as little pre-assumptions as possible.

Please don't hard code any IP in the source code of the Agent.